perun-rpc/src/main/java/cz/metacentrum/perun/rpc/ApiCaller.java

```diff
@@ -423,6 +423,10 @@ public Destination getDestination(String destination, String type, String propag
 		return d;
 	}
 
+	public Destination getDestinationById(int id) throws PerunException {
+		return getServicesManager().getDestinationById(rpcSession, id);
```

Comment:
i think you'll need to add RPC role to the method's perun-roles policies so that it does not fail with privilege exception